pkg/daemon/ceph/client/info.go

```diff
@@ -45,6 +45,7 @@ type ClusterInfo struct {
 	// If the CR name is needed, access it through the NamespacedName() method.
 	name              string
 	OsdUpgradeTimeout time.Duration
+	NetworkSpec       cephv1.NetworkSpec
```

Comment:
I'm going back and forth considering if this should just be a setting like:
```go
ExecuteRBDRemotely bool
```

But I can see how the network spec may be useful in the future if there are other places we need the network settings, so let's go with the the network spec as you have it.

Reply:
Also `ExecuteRBDRemotely` would be too restrictive and won't reflect well with the type's name.